core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -281,9 +281,13 @@ private[spark] object TestUtils {
     attempt.isSuccess && attempt.get == 0
   }
 
-  def isPythonVersionAtLeast38(): Boolean = {
+  def isPythonVersionAtLeast37(): Boolean = isPythonVersionAtLeast(3, 7, 0)
+
+  def isPythonVersionAtLeast38(): Boolean = isPythonVersionAtLeast(3, 8, 0)
```

Comment:
Let's remove if this isn't used now in the codebase

Reply:
[21ce6c7](https://github.com/apache/spark/pull/37487/commits/21ce6c7fbaf51caa52d00c19e197600c574cd69a) remove this

Reply:
@HyukjinKwon just have a question but not related to this pr

https://github.com/apache/spark/blob/1a1e4938f97ecedbb99a440df27d365439a9a621/sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala#L108

Should the above code call `TestUtils.isPythonVersionAtLeast37()` instead? like 

```
private lazy val isPythonAvailable: Boolean = TestUtils.isPythonVersionAtLeast37()
```